<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T2Cartas</class>
 <widget class="QDialog" name="T2Cartas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tValor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwOriginal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwNew">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bQuitar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitar Duplicados</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
